@xen-orchestra/lite/src/components/Console.tsx

```diff
@@ -52,7 +69,8 @@ const Console = withState<State, Props, Effects, Computed, ParentState, ParentEf
         // eslint-disable-next-line no-new
         new RFB(this.state.container.current, url, {
           wsProtocols: ['binary'],
-        })
+        }).addEventListener('disconnect', this.effects._attemptToReconnect)
+        this.state.isConnected = true
```

Comment:
The RFB never fails to be instantiated so `isConnected` will always be set to `true` here. However, the `disconnect` event is fired when the RFB fails to connect, even if it's very early, so simply triggering a `_connect` when the event is fired should be enough.